components/coding/GoalsSectionComponent.tsx

```diff
@@ -77,7 +77,7 @@ export default function GoalsSection({
       </div>
     </ExpandableContainer>
   ) : userGoals.length <= 0 && inProfile ? (
-    <ExpandableContainer open={true} title={""}>
+    <ExpandableContainer open={true} title={"Goals"}>
```

Comment:
we should call the ExpandableContainer on the index page instead of within the component for all components - this way, each component is reusable across the app with or without the ExpandableContainer

this implementation is affecting the layout of the goal page:

<img width="1490" alt="Screen Shot 2023-01-09 at 2 19 23 PM" src="https://user-images.githubusercontent.com/98791180/211390018-376c8f86-bd07-40a2-a99f-eab1098288d8.png">

Reply:
removed the container from all the individual components and added them all to the profile page.  Great suggestion!